components/automate-cli/cmd/chef-automate/automateHAAwsDeployment.go

```diff
@@ -205,7 +213,10 @@ func (a *awsDeployment) validateConfigFields() *list.List {
 func (a *awsDeployment) validateEnvFields() *list.List {
 	errorList := list.New()
 	if len(a.config.Aws.Config.Profile) < 1 {
-		errorList.PushBack("Invalid or empty aws profile name")
+		isIAMRole, _ := a.isIamRolePresent()
```

Comment:
why have you ignored the error returned here ? @vviveksharma

Reply:
The place where this function is called there we are doing `errorList.PushBackList(a.validateEnvFields())` , if I introduce the error field then this will not be able to push the value in the errorList

Reply:
Updated the function and it return type as error and checking the logic around that